warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -1138,7 +1139,7 @@ private void writeStats(Path[] jobDirectories) throws IOException {
             rawFS.setConf(conf);
             CompressionCodec cc = new GzipCodec();
             CompressionType ct = CompressionType.BLOCK;
-            
+
```

Comment:
Seeing these whitespace changes makes me wonder if this branch has been built? Or was it previously wrong and this change fixed it?

Reply:
It must not have been after the most recent fixes. Re-running with tests now and will update when complete,